### PR TITLE
starship: Add version 0.26.4

### PR DESCRIPTION
**Pull request by matchai**
Adds starship to Scoop.
Please let me know if anything is missing from the manifest. 👍

### CODE REVIEW

File: bucket/starship.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "description": "The minimal, blazing fast, and extremely customizable prompt for any shell!",
+    "homepage": "http://starship.rs",
+    "license": "ISC",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/starship/starship/releases/download/v0.26.3/starship-x86_64-pc-windows-msvc.zip",
+            "hash": "6d7729acba118432501032876dbdb9ab0867921e5fa599e77aed1e709569981d"
+        }
+    },
+    "bin": "starship.exe",
```
Comment (Ash258) [inline bucket/starship.json]:
```suggestion
    "extract_dir": "target\\x86_64-pc-windows-msvc\\release",
    "bin": "starship.exe",
```

A

File: bucket/starship.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/starship/starship/releases/download/$version/starship-x86_64-pc-windows-msvc.zip"
```
Comment (r15ch13) [inline bucket/starship.json]:
```suggestion
                "url": "https://github.com/starship/starship/releases/download/v$version/starship-x86_64-pc-windows-msvc.zip"
```

File: bucket/starship.json (hunk 1 above)
Comment (matchai) [reply bucket/starship.json]: Good catch! It looks like an oversight in our automated release process.
I'll be fixing it and making a new release to address this.

File: bucket/starship.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "homepage": "https://starship.rs",
+    "license": "ISC",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/starship/starship/releases/download/v0.26.3/starship-x86_64-pc-windows-msvc.zip",
+            "hash": "6d7729acba118432501032876dbdb9ab0867921e5fa599e77aed1e709569981d"
+        }
+    },
+    "bin": "starship.exe",
+    "checkver": "github",
```
Comment (linsui) [inline bucket/starship.json]:
```suggestion
    "checkver": {
        "github": "https://github.com/starship/starship"
    }
```

Comment (matchai) [reply bucket/starship.json]: Good catch! 👍

File: bucket/starship.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/starship/starship/releases/download/v0.26.3/starship-x86_64-pc-windows-msvc.zip",
+            "hash": "6d7729acba118432501032876dbdb9ab0867921e5fa599e77aed1e709569981d"
+        }
+    },
+    "bin": "starship.exe",
+    "checkver": {
+        "github": "https://github.com/starship/starship"
+    }
```
Comment (linsui) [inline bucket/starship.json]:
```suggestion
    },
```

Sorry...

File: bucket/starship.json (hunk 1 above)
Comment (matchai) [reply bucket/starship.json]: The binary should no longer be nested within folders in the archive. 👍

**Comment by r15ch13** [comment]
Since you are the creator of this fine starship, I would like to ask if you could provide checksums for each release? A `.sha256` file for each binary or a text file containing all checksums + filename works.

The auto-update part could be changed to something like that:
```json
"autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/starship/starship/releases/download/v$version/starship-x86_64-pc-windows-msvc.zip"
            }
        },
        "hash": {
            "url": "$url.sha256"
        }
    }
```
or
```json
"autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/starship/starship/releases/download/v$version/starship-x86_64-pc-windows-msvc.zip"
            }
        },
        "hash": {
            "url": "$baseurl/checksums.txt"
        }
    }
```

**Comment by marblenix** [comment]
This is the sample shasum file I managed to come up with:

```
x86_64-apple-darwin SHA256(starship)= 02dc44f332b47ebfb2e51724733f46f69ec8ceab50a58fc62e847b25561ea36e
x86_64-pc-windows-msvc SHA256(starship.exe)= 682ae8b3b7fc889d1a134f2c07d71764c59e57dd19b138da4267bdd926ec2067
x86_64-unknown-linux-gnu SHA256(starship)= 868ba3a773e424785828e8468534825d5bf2bd4cbf46f094e259edd37863b025
x86_64-unknown-linux-musl SHA256(starship)= 60b6b6ec0c21616349f0d53489519208a34b5daa5367dd04b07c37cb83f250a8
```

please let me know if the format of the file will work with scoop or not.

**Pull request merged**